bot/controllers/userAdd.js

```diff
@@ -0,0 +1,28 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+
+class UserAdd extends BaseController {
+  constructor(message) {
+    super(message);
+
+    const controller = this;
+
+    this.commands = [
+      new Command(
+        '!resend',
```

Comment:
The command ```!resend``` doesn't seem to have enough connection or context to the action you are trying to build. consider other options as 'resend' is ambiguous.